README.md

```diff
@@ -370,6 +372,29 @@ let () =
   painIndexMap |> Hashtbl.iter (fun k v -> Js.log {j|key:$k, val:$v|j})
 ```
 
+###### Belt.HashMap.String
+
+Mutable, string key type, BuckleScript only
+
+```ml
+open Belt
+let painIndexMap = HashMap.String.make ~hintSize:10
+let () =
+  let open HashMap.String in
+    set painIndexMap "western paper wasp" 1.0;
+    set painIndexMap "yellowjacket" 2.0;
+    set painIndexMap "honey bee" 2.0;
+    set painIndexMap "red paper wasp" 3.0;
+    set painIndexMap "tarantula hawk" 4.0;
+    set painIndexMap "bullet ant" 4.0
+
+let () =
+  HashMap.String.set painIndexMap "bumble bee" 2.0
+
+let () =
+  painIndexMap |. HashMap.String.forEach (fun k  -> fun v  -> Js.log {j|key:$k, val:$v|j})
```

Comment:
It's ok. but why do you think fast pipe is confusing?

Reply:
There's already a well established pipe operator, not just in OCaml/Reason but in many other languages as well. The fast pipe isn't even language-specific but platform-specific. It won't work outside BuckleScript, which makes it unusable with anything written to target anything other than BuckleScript. But even within BuckleScript it's both confusing and painful to have to remember which convention and pipe operator you have to use with a given function, especially if you have to juggle both in the same "chain" of function calls. And for newcomers, which is the intended audience here, unfamiliarity makes this confusion much worse.

Of course it's really the bad design of `Belt` itself that's the root problem here and the fast pipe is mostly a workaround to make serious use more convenient, but we can't do much about that other than to not show it, which I don't think is very honest since it's still a real alternative. But we can at least avoid using cryptic operators that's highly likely to be unfamiliar and confusing to newcomers when it's largely unnecessary to demonstrate it.